scripts/sfpshow

```diff
@@ -322,7 +322,7 @@ class SFPShow(object):
             sorted_table_keys = natsorted(port_table_keys)
             for i in sorted_table_keys:
                 interface = re.split(':', i, maxsplit=1)[-1].strip()
-                if interface and interface.startswith(front_panel_prefix()) and not interface.startswith(backplane_prefix()):
+                if interface and interface.startswith(front_panel_prefix()) and not interface.startswith(backplane_prefix()) and not interface.startswith(inband_prefix()):
```

Comment:
I think right now, there is no need to check `not interface.startswith(backplane_prefix()) and not interface.startswith(inband_prefix())` since `front_panel_prefix` is different from the other prefixes. But it's safer in case in future the prefixes are changed. So approved.

Reply:
Thanks for approving. But we changed the prefix. As per the review comments during Recycle port HLD, we changed the Inband port prefix to "Ethernet-IB". With this, it is required to handle inband_prefix similar to backplane_prefix.